src/main/java/StringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,74 @@
+package StringCalculator;
+
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Queue;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static final Pattern CHECK_NUMBER_PATTERN = Pattern.compile("-?\\d+(\\.\\d+)?");
+
+    public static int calculate(String input) {
+        validate(input);
+        List<String> strings = Arrays.asList(input.split(" "));
+        Deque<Integer> operands = new LinkedList<>();
```

Comment:
```suggestion
        Deque<Number> operands = new LinkedList<>();
```

우리만의 숫자객체가 있었어도 좋을것 같네요~